### ISSUE TITLE
Unable to Install on Linux Mint 20/R 4.3.3 - 

### ISSUE THREAD

**Issue by alexandermichels** [r-spatial/classInt#47]
Hello, I know very little about R/Fortran, but I'm trying to install this package and run into the following issue:

```
> install.packages("classInt")
Installing package into ‘/home/dijkstra/R/x86_64-pc-linux-gnu-library/4.3’
(as ‘lib’ is unspecified)
trying URL 'https://cloud.r-project.org/src/contrib/classInt_0.4-10.tar.gz'
Content type 'application/x-gzip' length 440271 bytes (429 KB)
==================================================
downloaded 429 KB

* installing *source* package ‘classInt’ ...
** package ‘classInt’ successfully unpacked and MD5 sums checked
** using staged installation
** libs
using C compiler: ‘gcc (Ubuntu 9.4.0-1ubuntu1~20.04.2) 9.4.0’
using Fortran compiler: ‘GNU Fortran (Homebrew GCC 12.2.0) 12.2.0’
gfortran  -fpic  -g -O2 -fdebug-prefix-map=/build/r-base-CIJAF4/r-base-4.3.3=. -fstack-protector-strong  -c fish1.f -o fish1.o
as: unrecognized option '--gdwarf-5'
make: *** [/usr/lib/R/etc/Makeconf:214: fish1.o] Error 1
ERROR: compilation failed for package ‘classInt’
* removing ‘/home/dijkstra/R/x86_64-pc-linux-gnu-library/4.3/classInt’
Warning in install.packages :
  installation of package ‘classInt’ had non-zero exit status

The downloaded source packages are in
	‘/tmp/RtmpnkoyD8/downloaded_packages’
```

System information:

```
> sessionInfo()
R version 4.3.3 (2024-02-29)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Linux Mint 20

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.9.0 
LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.9.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8     LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8    LC_PAPER=en_US.UTF-8      
 [8] LC_NAME=C                  LC_ADDRESS=C               LC_TELEPHONE=C             LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

time zone: America/Chicago
tzcode source: system (glibc)

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] compiler_4.3.3 tools_4.3.3   
```

Any help is greatly appreciated!

**Comment by edzer** [comment]
Try installing the binary package, not the source package.

**Comment by alexandermichels** [comment]
I'm not entirely sure how to install the binary package (know very little about R), but I did find the solution in case others run into this. I have linuxbrew installed, the solution was to update brew and then run `brew link binutils --force`. [This StackOverflow provided the info](https://stackoverflow.com/questions/74000991/gcc-11-is-not-debugging-it-shows-as-unrecognized-option-gdwarf-5). Hope this helps others!

Apologies for wasting your time, I was working on this for an hour before I made the issue and happened to find the solution right after opening it.
